collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -38,12 +37,38 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>parchment</support>
+                        <extent>i (paper) + i (parchment) + i (medieval parchment) + 158 + ii (parchment) <dimensions type="leaf" unit="mm">
+                              <height>395</height>
+                              <width>305</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>i–ii, 1–161, in pencil.</foliation>
+                        <collation>Quire signatures XV–XVIII and XXII in red span the two main parts (fols. 91r, 99r, 107r, 116r, and 148r), suggesting that the volume was mainly composed of quaternions, of which three are missing at the beginning. One would expect the 15th quaternion to start on the 113th leaf, not the 90th, so about 24 leaves are apparently missing. <secFol>Et verum</secFol></collation>
```

Comment:
Can the collation be verified? If not, e.g. too tightly bound, I think it's worth saying so to potentially save someone a job later on. If it can be verified but there isn't time, we should say this in a comment to aid our future selves